components/x-audio/src/components/index.jsx

```diff
@@ -32,8 +34,9 @@ export const Audio = ({
 		{expanded && <div className={classNameMap('audio-player__info__image')}><img alt="dummy"/></div>}
 		<div className={classNameMap('audio-player__info__title')}>{title}</div>
 		<div className={classNameMap('audio-player__info__series-name')}>{expanded ? seriesName : `${seriesName}:`}</div>
-		{!expanded && loading ? null : <TimeRemaining currentTime={currentTime} duration={duration} expanded={expanded}/>}
-		{loading && <Loading expanded={expanded} />}
+		{!expanded && loading || error ? null : <TimeRemaining currentTime={currentTime} duration={duration} expanded={expanded}/>}
+		{!error && loading && <Loading expanded={expanded} />}
+		{error && <ErrorMessage />}
```

Comment:
This makes me wonder if we should split out the minimised and expanded views into separate components, and just pass them all the props. Perhaps something to consider after the release?

Reply:
would be easier to understand it